cypress/support/commands.ts

```diff
@@ -23,3 +23,9 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+// Ignoring uncaught exceptions since errors from external apps should not stop de workshop
+Cypress.on("uncaught:exception", (err, runnable) => {
+  // returning false here prevents Cypress from
+  // failing the test
+  return false;
```

Comment:
If you have this yo don't need the statement on the before

Reply:
I will fix it for the next PR.   Thanks for the advise.